cli/ioctl/cmd/action/action.go

```diff
@@ -159,6 +159,10 @@ func sendAction(elp action.Envelope, signer string) error {
 		log.L().Error("failed to sign action", zap.Error(err))
 		return err
 	}
+	err = isBalanceEnough(signer, sealed)
```

Comment:
The code style convention is: 

```suggestion
	if err := isBalanceEnough(signer, sealed); err != nil {
```